src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php

```diff
@@ -241,7 +241,7 @@ public function toArray()
             'action' => $this->getAction(),
             'display' => $this->getDisplay(),
             'component' => $this->getComponent(),
-            'componentOptions' => $this->getComponentOptions(),
+            'componentOptions' => array_merge($this->getComponentOptions(), $options),
```

Comment:
Generally yes, but in this case I am injecting more options to be merged here... I don't think I can add additional options using the serializer, or am I wrong?